README.md

```diff
@@ -105,6 +106,15 @@ Example command:
   node.js V16 and above
 ```
 
```

Comment:
Great job @gisachris 🤝

- [ ] [Optional] I would recommend you name the project `To-Do-List` in your readme file instead of `webpack template` so as to maintain the same project name.